app/cafe.py

```
@@ -0,0 +1,24 @@
+from datetime import date
+
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated.")
+
+        today = date.today()
+        expiration_date = visitor["vaccine"]["expiration_date"]
+        if expiration_date < today:
+            raise OutdatedVaccineError("Vaccine is expired.")
+
+        if visitor.get("wearing_a_mask") is not True:
+            raise NotWearingMaskError("NotWearingMaskError")

```

This violates checklist item 3: "Use descriptive error messages:". The NotWearingMaskError is raised with the message "NotWearingMaskError" which is not descriptive. Please raise the error with a clear message, for example: `NotWearingMaskError("Visitor is not wearing a mask.")`. See the raise site in visit_cafe.